lib/vmdb/settings/validator.rb

```diff
@@ -184,6 +195,16 @@ def smtp(data)
           errors << [:from, "\"#{data.from}\", invalid. Should be a valid email address"]
         end
 
+        if send_notifications && @roles.split(",").include?("notifier") && data.domain == "mydomain.com"
```

Comment:
We were just talking about how this mydomain.com is probably bad and we should switch to example.com.  I'm wondering if we should make it a constant somewhere that both sides can share, so we don't accidentally change one and not the other.

Reply:
Exactly, great point.  I was considering changing the domain but first wanted to see if something like this makes sense.  I'm not sure how else to check that smtp is configured than to look if the domain was changed.  I guess we could check other fields too.